pkg/node/node.go

```diff
@@ -100,6 +100,15 @@ func (a *Node) Serve(ctx context.Context) error {
 	}
 }
 
+func (a *Node) LogErrors(err error) {
```

Comment:
Definitely this is not a public function. May be there is no need for a separate function at all if it used once...

Reply:
You're right but the function "Run" is big enough I'll make that method private